composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "apy/datagrid-bundle",
+    "name": "artscorestudio/datagrid-bundle",
```

Comment:
I noticed you changed the repository name, which should not be included in this PR.